syntax = "proto3";
package rj_logread;

message LogReadRequest {
    bytes node_name = 1;
    bytes time = 2; //格式为%4d-%2d-%2d
}

message LogReadResponse {
    int32 serial = 1;   //序号
    bool is_end = 2;    //是否结束了
    bytes msg = 3;
}